modules/402-ingress-nginx/templates/kruise/webhook.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: kruise-mutating-webhook-configuration
+  {{- include "helm_lib_module_labels" (list . ) | nindent 2 }}
+  annotations:
+    template: ""
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: kruise-webhook-service
+        namespace: d8-ingress-nginx
+        path: /mutate-apps-kruise-io-v1alpha1-daemonset
+    failurePolicy: Fail
+    admissionReviewVersions:
+      - v1
+    sideEffects: None
+    timeoutSeconds: 30
+    name: mdaemonset.kb.io
+    rules:
+      - apiGroups:
+          - apps.kruise.io
+        apiVersions:
+          - v1alpha1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - daemonsets
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: kruise-webhook-service
+        namespace: d8-ingress-nginx
+        path: /mutate-pod
+    failurePolicy: Ignore
+    admissionReviewVersions:
+      - v1
+    sideEffects: None
+    timeoutSeconds: 30
+    name: mpod.kb.io
+    namespaceSelector:
+      matchExpressions:
+        - key: control-plane
+          operator: DoesNotExist
+    rules:
+      - apiGroups:
+          - ""
+        apiVersions:
+          - v1
+        operations:
+          - CREATE
+        resources:
+          - pods
+---
+# Source: kruise/templates/webhookconfiguration.yaml
```

Comment:
```suggestion
# Source: kruise/templates/webhookconfiguration.yaml
```
Couldn't find where this file is.

Reply:
It's an official kruise helm chart templates. I left these comment to easily figure out where it came from